.github/workflows/test.yml

```diff
@@ -8,12 +8,21 @@ on:
   workflow_dispatch:
 
 env:
-  TOOLCHAIN_VERSION: 22
+  TOOLCHAIN_VERSION: 23
+  GRADLE_JAVA_VERSION: 23
   CLANG_LLVM_VERSION: 13.0.0
   CLANG_LLVM_BASE_URL: "https://github.com/llvm/llvm-project/releases/download/llvmorg-13.0.0/"
   JTREG_VERSION: 7.3.1+1
 
 jobs:
+  validate-gradle-wrapper:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4.1.1
+        with:
+          fetch-depth: 1
+      - uses: gradle/wrapper-validation-action@v3
```

Comment:
`gradle/wrapper-validation-action` is deprecated, and this causes a warning now, see for example https://github.com/JornVernee/jextract/actions/runs/12632919612/job/35197503582#annotation:3:26
> Warning: This job uses deprecated functionality from the 'gradle/wrapper-validation-action' action. Consult the logs for more details.
> DEPRECATION: The action `gradle/wrapper-validation-action` has been replaced by `gradle/actions/wrapper-validation`. See https://github.com/gradle/actions/blob/main/docs/deprecation-upgrade-guide.md#the-action-gradlewrapper-validation-action-has-been-replaced-by-gradleactionswrapper-validation

Reply:
Oh, I saw that as well, and thought I grabbed the non-deprecated one.